.circleci/config.yml

```diff
@@ -171,17 +183,29 @@ references:
             echo "$(date +%s)-$(cat .version)-dev-$(git rev-parse --short HEAD)" > .version
             cat .version
             ytool -s appVersion "$(cat .version)" -s version "$(cat .version)" -f "./codacy/Chart.yaml" -e
+      - run:
+          name: Set environment specific helm values
+          command: |
+            # DELETE ME: this is a temporary fix because breaking changes were added to ci-do so we can't add gettext
+            apk add -U gettext
+            ./.circleci/setupEnvironmentVars.sh .do/values-env-temp.yaml > ./.do/values-env.yaml
       - deploy:
           name: Install Codacy
-          command: make -e -C .do/ deploy_to_doks_from_local_dir
+          command: make -e -C .do/ deploy_to_doks_from_local_dir HELM_INSTALL_EXTRA_FLAGS="-f values-env.yaml"
 
   deploy_to_cluster_from_chartmuseum: &deploy_to_cluster_from_chartmuseum
     steps:
       - <<: *attach_workspace
       - <<: *doctl_authenticate
+      - run:
+          name: Set environment specific helm values
+          command: |
+            # DELETE ME: this is a temporary fix because breaking changes were added to ci-do so we can't add gettext
```

Comment:
When could we delete this?

Reply:
When we have everything with helm3 so that the package can be added to ci-do, or if we decide to do a more piecewise migration and have helm2 and helm3 side by side for a while.